config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
+
   scope "(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
     root "pages#index"
     get "/home", to: "pages#index"
+    get "/signup", to: "users#new"
+    post "/signup", to: "users#create"
+
+    resources :users, only: %i(new create show edit update)
```

Comment:
signup thì chưa cần dùng đến toàn bộ resource đúng k? vậy bỏ đi những resources k cần thiết nhé

Reply:
Oki chị ;)